docs/examples/coreos-kops-tests-multimaster.md

```diff
@@ -0,0 +1,310 @@
+||  CreationDate       |  2017-08-10T02:07:16.000Z                      ||
+||  Description        |  CoreOS Container Linux stable 1409.8.0 (HVM)  ||
+||  EnaSupport         |  True                                          ||
+||  Hypervisor         |  xen                                           ||
+||  ImageId            |  ami-32705b49                                  ||
+||  ImageLocation      |  595879546273/CoreOS-stable-1409.8.0-hvm       ||
+||  ImageType          |  machine                                       ||
+||  Name               |  CoreOS-stable-1409.8.0-hvm                    ||
+||  OwnerId            |  595879546273                                  ||
+||  Public             |  True                                          ||
+||  RootDeviceName     |  /dev/xvda                                     ||
+||  RootDeviceType     |  ebs                                           ||
+||  SriovNetSupport    |  simple                                        ||
+||  State              |  available                                     ||
+||  VirtualizationType |  hvm                                           ||
+|+---------------------+------------------------------------------------+|
+|||                         BlockDeviceMappings                        |||
+||+-----------------------------------+--------------------------------+||
+|||  DeviceName                       |  /dev/xvda                     |||
+|||  VirtualName                      |                                |||
+||+-----------------------------------+--------------------------------+||
+||||                                Ebs                               ||||
+|||+------------------------------+-----------------------------------+|||
+||||  DeleteOnTermination         |  True                             ||||
+||||  Encrypted                   |  False                            ||||
+||||  SnapshotId                  |  snap-00d2949d7084cd408           ||||
+||||  VolumeSize                  |  8                                ||||
+||||  VolumeType                  |  standard                         ||||
+|||+------------------------------+-----------------------------------+|||
+|||                         BlockDeviceMappings                        |||
+||+----------------------------------+---------------------------------+||
+|||  DeviceName                      |  /dev/xvdb                      |||
+|||  VirtualName                     |  ephemeral0                     |||
+||+----------------------------------+---------------------------------+||
+```
+
+Also, you can obtaing the owner/name using the command (that is what we really need for KOPS):
+
+```bash
+aws ec2 describe-images --region=us-east-1 --owner=595879546273 \
+    --filters "Name=virtualization-type,Values=hvm" "Name=name,Values=CoreOS-stable*" \
+    --query 'sort_by(Images,&CreationDate)[-1].{id:ImageLocation}' \
+	--output table
+
```

Comment:
Given this command doesn't use the AMI we just found here?  Or should we remove the previous way of querying?  I worry it's confusing to find an AMI using the CoreOS JSON file, but then we don't use it...

I do like your way though!